dali/operators/math/expressions/arithmetic_meta.h

```diff
@@ -73,8 +96,28 @@ DALI_HOST_DEV constexpr int GetOpArity(ArithmeticOp op) {
   switch (op) {
     case ArithmeticOp::plus:
     case ArithmeticOp::minus:
+    case ArithmeticOp::sqrt:
```

Comment:
How about having `rsqrt`? Not that it matters much for GPU (it's bandwidth limited), but CPU could definitely use it.

Reply:
Done